src/python/pants/option/global_options.py

```diff
@@ -35,7 +35,7 @@
 from pants.option.scope import GLOBAL_SCOPE
 from pants.option.subsystem import Subsystem
 from pants.util.dirutil import fast_relpath_optional
-from pants.util.docutil import doc_url
+from pants.util.docutil import bin_name, doc_url
```

Comment:
Note that a few (2 maybe) of the bootstrap options use `./pants` in their strings. It'd be a chicken-and-egg for us to try and replace those :smile: 

I think if help messages were declared lazily we could "get this right" (it'd also fix the import ordering issue). But for such a small feature it isn't worth it :smile: